README.md

```diff
@@ -5,6 +5,9 @@ cluster inside a VM. You can try out OpenShift or develop with it, day-to-day, o
 
 Minishift uses [libmachine](https://github.com/docker/machine/tree/master/libmachine) for
 provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) for running the cluster.
+For general information about OpenShift cluster architecture,  
```

Comment:
I don't think the leading paragraphs are a good place to add such a link.
+1, maybe we could add this link towards the end of the Quickstart section.